monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -80,43 +99,50 @@ class AppComponent extends AuthComponent {
     }
 
     if (res) {
+      // TODO: get plugins only one time
```

Comment:
Looks like this TODO isn't needed

Reply:
I haven't run a check for this behavior yet

Reply:
What's the point of `pluginsFetched`?

Reply:
I changed the variable name from 'fetched'.